metadata_tools/geometry_support.py

```
@@ -627,81 +876,106 @@ def _construct_excluded_mask(backplane, target, primary, mask_desc, *,
-                results = ['"'+str(s[0])+'"', '"'+str(s[1])+'"']
+                results = [str(s[0]), str(s[1])]
 
         # Write the formatted value(s)
         strings = []
         for number in results:
             error_message = ""
 
+            # numeric values: flag common exceptions and use standard format
             if not isinstance(number, str):
                 if np.isnan(number):
                     warnings.warn("NaN encountered")
```

Ruff (0.14.10)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)